content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -1,29 +1,26 @@
 ---
-title: Role-Based Access Control
+title: Authorization Policy
 description: Shows how to set up role-based access control for services in Istio mesh.
 weight: 40
-keywords: [security,access-control,rbac]
+keywords: [security,access-control,rbac,authorization]
 ---
 
-This task shows how to set up role-based access control (RBAC) for services in Istio mesh. You can read more about Istio
-RBAC from [Istio RBAC concept page](/docs/concepts/security/#role-based-access-control-rbac).
+This guide covers the main tasks you might need to perform to set up Istio authorization, also known
+as Istio Role Based Access Control (RBAC), for services in an Istio mesh. You can read more in
+[Istio authorization](/docs/concepts/security/#role-based-access-control-rbac).
 
 ## Before you begin
 
-* Set up Istio on auth-enabled Kubernetes by following the instructions in the
+The tasks in this guide assume that you:
+
+* Understand [Istio authorization](/docs/concepts/security/#role-based-access-control-rbac) concepts.
+
+* Have set up Istio on Kubernetes **with authentication enabled** by following the instructions in the
   [quick start](/docs/setup/kubernetes/quick-start/).
```

Comment:
Istio Authentication requires mutual TLS to work. Authentication should be enabled in the installation steps.

Reply:
Done. Just curious I thought we could also use JWT-based authentication and in this case mTLS is optional?

Reply:
How about change it to: This tutorial requires mutual TLS to work. Mutual TLS authentication should be enabled in the installation steps.

Reply:
Done.